packages/components/src/Avatar/Avatar.mdx

```diff
@@ -0,0 +1,94 @@
+---
+name: Avatar
+menu: Components
+route: /components/avatar
+---
+
+import { useState } from "react";
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Avatar } from ".";
+
+# Avatar
+
+<ComponentStatus stage="pre" responsive="no" accessible="yes" />
+
+An avatar will be used to display an visual identifier for an individual user.
+
+```ts
+import { Avatar } from "@jobber/components/Avatar";
+```
+
+<Playground>
+  <Avatar
+    imageUrl="https://api.adorable.io/avatars/150/jobbler"
+    name="The Jobbler"
+    size="large"
+    initials="JBLR"
+    color="var(--color-indigo)"
+  />
+</Playground>
+
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Avatar component consider the
```

Comment:
I think it would be good to inline some of the usage guidelines from the file here. When is it appropriate to use the component. What is the intention of color. 

Some ideas here: https://polaris.shopify.com/components/images-and-icons/avatar

Reply:
Like this!

Let's pull the figma link out for the moment. We should do some circling around on when is appropriate to link into other things. 😄